commonui/src/main/java/com/hyperwallet/android/ui/common/view/HorizontalDividerItemDecorator.java

```diff
@@ -33,9 +33,13 @@
     private static final short DEFAULT_PADDING = 0;
 
     protected final Drawable mHorizontalItemDivider;
+    private final boolean mHasTopDivider;
+    private final float topDividerPadding;
```

Comment:
Suggestion: i think naming may be for this variable **mTopDividerPadding**.

Reply:
👍 thanks did not see that!